neptune/new/internal/backends/swagger_client_wrapper.py

```diff
@@ -35,22 +33,22 @@ def __init__(self, api_method):
         self._api_method = api_method
 
     @staticmethod
-    def _parse_error_type(response) -> Optional[str]:
+    def handle_neptune_http_errors(response, exception: Optional[HTTPError] = None):
         try:
-            error_data = json.loads(response.text)
-            return error_data.get("errorType") if error_data is not None else None
-        except JSONDecodeError:
-            return None
+            body = response.json() or dict()
```

Comment:
Will `json.loads(response.text)` still work, or?

* we didn't release client with this code yet, so no problem
* we don't care - we'll talk to clients to upgrade client.

Reply:
Both will work and we didn't release it anyway.